### ISSUE TITLE
Unsure How to Update Game Request

### ISSUE THREAD

**Issue by andyjzhong** [ga-wdi-boston/game-project#623]
I have the authentication request working and I have the game logic working but I need assistance with updating the Game API with gameplay.

I know I need to create a game within the Game API and log the X's and O's into each new game. I think it has something to do with making POST requests but I'm not sure how to do it. I haven't tried anything with this yet but at a loss for where to begin.

**Comment by andyjzhong** [comment]
Woot! I think I'm one step closer!

So I reviewed the Game-Project-API and searched past issues and found that Matt's Issue #536 was able to help me create a new game. Below is the code for my approach.

HTML Button:
```html
<form id="createGame">
  <input class="btn btn-lg btn-danger reset-button" type="submit" name="submit" value="New Create Game">
</form>
```

Event Listener in Events.js:
```js
// Trying to create a new game.
const onCreateGame = function (event) {
  event.preventDefault()
  console.log('onCreateGame from events.js ran!')
  api.createGame()
    .then(ui.onNoContentSuccess)
    .catch(ui.onError)
}
```

AJAX request in Api.js:
```js
// Creates the Ajax function to POST data for new game!
const createGame = (data) => {
  console.log('createGame from api.js ran!')
  return $.ajax({
    url: config.apiOrigin + '/games/',
    method: 'POST',
    headers: {
      Authorization: 'Token token=' + store.user.token
    }
  })
}
```

Ui Success/Fail Notification:
```js
const createGameSuccess = (data) => {
  console.log('createGameSuccess from ui.js ran!')
}

const createGameFailure = (error) => {
  console.error(error)
}
```

Both console.logs indicate that I've successfully created the game and I even received a "Completed 201" message in my API terminal. However, I'm not sure if I'm supposed to still see an "HTTP Token: Access denied" message on my localhost:4741/games -- I guess I would have expected to see some preliminary game objects with IDs in it. Can someone please help verify if I'm on the right track? I feel that I may need some assistance with putting in clicked X's and O's into these created games later, but that may have to be a separate issue request.

Thank you!!

**Comment by jordanallain** [comment]
what if you console.log(data) in your createGameSuccess function?

**Comment by andyjzhong** [comment]
Thanks Jordan -- I'll give this a shot tonight and let you know what I find.

**Comment by benjimelito** [comment]
Any updates on this?

**Comment by andyjzhong** [comment]
Hi Ben -- Not yet, I'll revisit this after class today.

**Comment by andyjzhong** [comment]
Hi All -- I tried the console.log(data) in my createGameSuccess function and it looks like I was able to successfully create a game! My console returns a game and game id with an empty array. My next step is adding the ability to log the X's and O's into the actual game objects. I know I need to use some sort of Patch request so I will give that a shot tonight and create a new issue log if necessary.
